src/components/Progress/Progress.scss

```diff
@@ -63,28 +71,35 @@ $progress-xs-height: 4px;
 
         transition: transform 0.6s ease, width 0.6s ease, background-color 0.6s ease;
 
-        &_theme_default {
-            background-color: var(--g-color-base-neutral-medium);
-        }
+        background-color: var(
+            --g-progress-filled-background-color,
+            var(--_--filled-background-color)
+        );
 
-        &_theme_success {
-            background-color: var(--g-color-base-positive-medium);
-        }
+        &_theme {
+            &_default {
```

Comment:
Lets write this styles in more flatten way. For example `&_theme_default` and so on